cms/models.py

```diff
@@ -54,10 +80,13 @@ def get_context(self, request, *args, **kwargs):
             "js_settings_json": json.dumps(_serialize_js_settings(request)),
             "site_name": settings.SITE_NAME,
             "title": self.title,
+            "program_description_section": self.program_description_section,
         }
 
+    subpage_types = ["ProgramDescriptionPage"]
```

Comment:
why not `ThreeColumnImageTextPage ` and `LearningResourcePage ` are provided as the subpage_types in it?

Reply:
Please add these two in subpage_types.

Reply:
Not need for this PR. When we will work over these two pages for `HomePage` then add accordingly.